### DISCUSSION TITLE
Problem loading local resources in Apache

### DISCUSSION THREAD

**Discussion by ejgutierrez74** [leokhoa/laragon#596] [Q&A]
Hi, ive downloaded and installed latest Laragon 6.0 Full Edition. For permission problems i had to install it in D: ( Windows 10 system), instead of default C:/.
So something as easy as put an image in a html code:  

`<img src="img/img_girl.jpg" alt="Girl in a jacket" width="500" height="600"> `

So in local ( open the html file in firefox for example, without using server) it works fine ( so the code is right and the path to the image).

But when i start apache and go to localhost in my browser...the image is not shown.... It appears and strange Apache error message ( something like Not allowed to load local resource). I didnt find something like this in my programming experience.

Other mates, have no problem if they install laragon in C:/ ( the ones who have its own computer and dont have any permission issue).

Seems like a bug, or something with laragon ( apache is the only service opened) in D:/

Another clue, is that if we use the live server extension in visual studio code, it works...

The last clue, i tried to open/view the file in chrome but also in firefox and edge...so some issues with chrome security settings can be discarded, in my opinion

Just in case, we are using laragon ( apache) in localhost, just to teach html and css. Client Browser is also in the same machine...

Any help or ideas apreciated.

Thanks for your work

**Comment by Pen-y-Fan** [comment]
My blog on permissions may help: [How to set up Laragon on a new Windows computer (part 2) - permissions](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-2/)

**Reply by ejgutierrez74** [reply]
id try, but i only use apache, i launch it, and it works fine ( laragon default web appears) and when i go to localhost/myweb ...the index.html page is shown...but the images dont. Saying the above error....
Thanks for your help....

**Reply by Pen-y-Fan** [reply]
Laragon uses [pretty urls](https://laragon.org/docs/pretty-urls.html) for each project, instead of localhost/myweb try http://myweb.test
